web/rootfs/etc/cont-init.d/10-config

```diff
@@ -132,6 +132,14 @@ if [[ ! -f /config/config.js ]]; then
         -e "s#// resolution:.*#resolution: 480,#" \
         /config/config.js
 
+    sed -i \
+        "s/stun:meet-jit-si-turnrelay.jitsi.net:443/${JITSI_WEB_STUN_SERVER}/g" \
```

Comment:
Can you rewrite matching so that it won't be globally only for one line, please look at the upper matching, is it possible?